mfr/server/app.py

```diff
@@ -36,8 +35,7 @@ def stop_loop():
 def make_app(debug):
     app = tornado.web.Application(
         [
-            (r'/static/(.*)', tornado.web.StaticFileHandler, {'path': server_settings.STATIC_PATH}),
-            (r'/assets/(.*?)/(.*\..*)', ExtensionsStaticFileHandler),
+            (r'/assets/(.*)', tornado.web.StaticFileHandler, {'path': server_settings.STATIC_PATH}),
```

Comment:
Dont need to have as a strict regex here anymore to prevent people from requesting source files and stuff - everything gets served out of dist.
There may be value in only using this handler for development and putting the static assets on cdn.osf.io or whatever the domain we use it.